examples/sprite_animation/main.rs

```diff
@@ -136,20 +166,11 @@ fn main() -> amethyst::Result<()> {
     let assets_dir = app_root.join("examples/sprite_animation/assets/");
     let display_config_path = app_root.join("examples/sprite_animation/config/display.ron");
 
-    let mut game_data = DispatcherBuilder::default()
-        .with_system_desc(
-            PrefabLoaderSystemDesc::<MyPrefabData>::default(),
-            "scene_loader",
-            &[],
-        )
-        .add_bundle(AnimationBundle::<AnimationId, SpriteRender>::new(
-            "sprite_animation_control",
-            "sprite_sampler_interpolation",
-        ))?
-        .add_bundle(
-            TransformBundle::new()
-                .with_dep(&["sprite_animation_control", "sprite_sampler_interpolation"]),
-        )?
+    let mut game_data = DispatcherBuilder::default();
+    game_data
+        .add_bundle(LoaderBundle)
+        .add_bundle(AnimationBundle::<AnimationId, SpriteRender>::default())
+        .add_bundle(TransformBundle::default())
```

Comment:
As we use the `CommandBuffer` in the animation systems, do we get a 1 frame delay here?
The cmd buffer is not flushed after each system run, right? So we need to manually invoke a flush before the Transform or Rendering Bundle.

Reply:
added a flush for safety, although maybe should be inside AnimationBundle.  let's see what people report.

we can add some slow down functionality in the example in the future to demonstrate features of animation or something like that which will let us debug further if there is an issue.